### DISCUSSION TITLE
GitHub Code Search have unexpected behaviour in High contrast dark theme

### DISCUSSION THREAD

**Discussion by wasup-yash** [community/community#57747] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

In GitHub's `high contrast dark theme`, when you type text in the search box and select it, it will be completely hidden with a blue box kind of bar which is unexpected as it is not visible for what are the selected words. it only `happens with the search box` the navigation search box which is on the 
![github](https://github.com/community/community/assets/112958086/3cfdc854-a114-4e66-b1b3-bc02f898ddad)
top left corner to be precise.

**Comment by cehfisher** [comment, marked as answer]
Hello @wasup-yash - thank you for providing this feedback! After reviewing your issue report against the latest UI, this issue appears to be resolved now. Could you please confirm and close this issue? Thank you!
<img width="292" alt="Screenshot highlighting a word in the search bar." src="https://github.com/community/community/assets/15240149/2d498185-a62a-4cbe-bda6-0b6cac0dcca5">

**Reply by wasup-yash** [reply]
yes with the new updated UI, it is resolved. thank you
